tsconfig-strict.json

```diff
@@ -134,6 +134,7 @@
     "core/templates/domain/collection/guest-collection-progress.service.spec.ts",
     "core/templates/domain/collection/search-explorations-backend-api.service.ts",
     "core/templates/domain/collection/search-explorations-backend-api.service.spec.ts",
+    "core/templates/domain/editor/undo_redo/base-undo-redo.service.ts",
```

Comment:
Please also type the spec file for this service.

Reply:
Actually, this service doesn't have a .spec file.

Reply:
Ah, okay.